### PR TITLE
Fix #2521

### LINKED ISSUE #2521: podman stop: timeouts broken again
This is essentially a repeat of  #2472

pr #2364 seems to have misdone a conflict resolution; it broke #2476.

### PR DESCRIPTION
**Pull request by jwhonce**
* Bad merge against podman stop, restored overwritten code

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

**Comment by rhatdan** [comment]
LGTM
@mheon @edsantiago PTAL

**Comment by mheon** [comment]
Lint is angry. Appease the CI gods and I will LGTM.

**Comment by edsantiago** [comment]
Tests pass again. Thank you.

File: test/e2e/stop_test.go [hunk 1]
```diff
@@ -107,6 +107,32 @@ var _ = Describe("Podman stop", func() {
...
+		Expect(session.ExitCode()).To(Equal(0))
+		cid1 := session.OutputToString()
+
+		session = podmanTest.Podman([]string{"stop", "--timeout", "1", "test5"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		output := session.OutputToString()
+		Expect(output).To(ContainSubstring(cid1))
+	})
+
```
Comment (edsantiago) [inline test/e2e/stop_test.go]: I'm not sure any of these tests would've caught the problem: that is, the problem is that the container is killed immediately, without a chance to trap SIGTERM. Could you try running these against a broken build? If I read them correctly, they will all pass.

**Comment by mheon** [comment]
I'm going to slap an approve on here in the interest of getting 1.1.2 out the door today, but more tests to address the issue @edsantiago brought up would be appreciated
/approve

**Pull request merged**